Source/Core/DolphinQt2/Config/Graphics/SoftwareRendererWidget.cpp

```diff
@@ -135,6 +135,12 @@ void SoftwareRendererWidget::SaveSettings()
       if (backend_name != SConfig::GetInstance().m_strVideoBackend)
       {
         SConfig::GetInstance().m_strVideoBackend = backend_name;
+
+        g_Config.Refresh();
```

Comment:
Should this be done in BackendChanged instead? This is identical to what you have in GeneralWidget::SaveSettings.